tests/designs/vhdl_zerovector/Makefile

```
@@ -0,0 +1,32 @@
+# Copyright cocotb contributors
+# Licensed under the Revised BSD License, see LICENSE for details.
+# SPDX-License-Identifier: BSD-3-Clause
+#
+# Simulator (modelsim is still 32-bit only)
+SIM ?= modelsim
+ifneq ($(filter $(SIM), modelsim),)
+    ARCH ?= i686
+endif
+
+# Language of toplevel module
+TOPLEVEL_LANG = vhdl
+
+ifneq ($(filter $(SIM), questa modelsim),)
+    # VHDL language version
+    VCOM_ARGS += -2008
+    # simulation time precision
+    SIM_ARGS += -t ns
+endif
+
```

You (accidentally?) removed this, please keep it,  otherwise Xcelium will not compile the file.
```suggestion
ifneq ($(filter $(SIM),ius xcelium),)
    SIM_ARGS += -v93
endif

```

---

The removal was not by accident. As I was not able to test it, I thought it would be better without. Didn't know that it was nessesary for the CI. Should be back with the fixup

---

Not strictly needed for CI (we don't have a license for Xcelium), but anyone who wants to try it locally (as I did).